portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -28,16 +28,24 @@
 public class DataServlet extends HttpServlet {
 
   static final String CONTENT_TYPE = "application/json;";
+  final List messages = new ArrayList<String>();
```

Comment:
Please see type inference comment from another PR. Prefer to fix here.

Reply:
Should be fixed!